monai/data/utils.py

```diff
@@ -983,3 +988,56 @@ def sorted_dict(item, key=None, reverse=False):
     if not isinstance(item, dict):
         return item
     return {k: sorted_dict(v) if isinstance(v, dict) else v for k, v in sorted(item.items(), key=key, reverse=reverse)}
+
+
+def convert_tables_to_dicts(
+    dfs,
+    row_indices: Optional[Sequence[Union[int, str]]] = None,
+    col_names: Optional[Sequence[str]] = None,
+    col_groups: Optional[Dict[str, Sequence[str]]] = None,
```

Comment:
we need some basic support for setting defaults for the missing values and also datatypes (as I mentioned in this example https://github.com/Project-MONAI/MONAI/issues/2310#issuecomment-864000564).

Reply:
Hi @wyli ,

Sorry I misunderstood your point yesterday, do you mean to add another arg `column_types` to translate loaded values?
```
column_types = {  # this could come from some json-based mapping file https://www.w3.org/2013/csvw/wiki/CSV-LD)
    "subject_id": {"type": str}, 
    "label": {"type": int, "default": 0},
    "ehr_0": {"type": float, "default": 0.0, "unit": "year"},
    "ehr_1": {"type": float, "default": 0.0, "unit": "month"},
    "image": {"type": nifti, "default": None},
}
```

Reply:
Hi @wyli ,

I added support for `type` and `default`, not sure how to use `unit` to handle data so I didn't add it.
Maybe we can add it later if user requests it.

Thanks.